tests/torch/quantization/test_functions.py

```diff
@@ -367,3 +368,34 @@ def test_quantize_asymmetric_backward(self, _seed, input_size, bits, use_cuda, i
             test_grads = get_grads([test_input, test_input_low, test_input_range])
 
             check_outputs_for_quantization_functions(test_grads, ref_grads, is_fp16)
+
+
+@pytest.mark.parametrize('quantization_mode', ['symmetric', 'asymmetric'])
+@pytest.mark.parametrize('device', ['cuda', 'cpu'])
+def test_mapping_to_zero(quantization_mode, device):
+    x_zero = torch.zeros([1]).to(torch.device(device))
+    levels = 256
+    eps = 1e-6
+    number_of_samples = 100
+    from torch.distributions.uniform import Uniform
```

Comment:
You forgot to set up an exact seed for this test, making it indeterministic, which is not convenient.